fdbclient/include/fdbclient/GetEncryptCipherKeys_impl.actor.h

```diff
@@ -39,6 +39,29 @@
 
 #define DEBUG_GET_CIPHER false
 
+namespace {
+ACTOR template <class T>
```

Comment:
Not for this PR, but a followup improvement (post 71.3.5 for sure):

`monitorGetEncryptCipherKeys` and `GetEncryptCipherKeysMonitor` can be replaced with an actor that waits on a delay, then waits on never, and it increments the counter after the delay, and decrements the count when cancelled.

Reply:
Can you give an example of what you mean? I think the code is doing essentially what you describe right now.